apps/mudita-center-e2e/wdio.conf.ts

```diff
@@ -55,29 +55,7 @@ export const config: Options.Testrunner = {
   // then the current working directory is where your `package.json` resides, so `wdio`
   // will be called from there.
   //
-  specs: [
-    toRelativePath(TestFilesPaths.displayInitialOsVersionTest),
-    toRelativePath(TestFilesPaths.checkForUpdateTest),
-    toRelativePath(TestFilesPaths.deviceUpdateTest),
-    toRelativePath(TestFilesPaths.mcVersionCheckTest),
-    toRelativePath(TestFilesPaths.contactsListTest),
-    toRelativePath(TestFilesPaths.contactsInAppNavigationTest),
-    toRelativePath(TestFilesPaths.messagesInAppNavigationTest),
-    toRelativePath(TestFilesPaths.newsInAppNavigationTest),
-    toRelativePath(TestFilesPaths.overviewInAppNavigationTest),
-    toRelativePath(TestFilesPaths.settingsInAppNavigationTest),
-    toRelativePath(TestFilesPaths.helpWindowCheckTest),
-    toRelativePath(TestFilesPaths.sarWindowCheckTest),
-    toRelativePath(TestFilesPaths.tosPrivacyLicenceWindowsCheckTest),
-    toRelativePath(TestFilesPaths.messageSendTest),
-  ],
-  suites: {
```

Comment:
You deleted main functionality of automation which is suites for testing various things:

- smoke test set
- mc standalone tests
- harmony only tests
- pure only tests
- kompakt only tests 
- multidevice tests

Reply:
I need to divide the test set into those that run locally vs. those that are to be run on CI/CD.

The file can support suites, if you want, it's not a problem. Do you already have some suites defined so that I can take them into account?

Reply:
https://github.com/mudita/mudita-center/pull/1763/commits/f510db0579f2e85c73ada897ab6bfaf043cb0b74